WordPressKit/WordPressOrgRestApi.swift

```diff
@@ -68,7 +68,7 @@ public final class WordPressOrgRestApi: NSObject {
         if let userAgent {
             additionalHeaders["User-Agent"] = userAgent
         }
-        if case let Site.dotCom(siteID: _, bearerToken: token) = site {
+        if case let Site.dotCom(siteID: _, bearerToken: token, _) = site {
```

Comment:
Nitpick: The new `_` entry made me realize that the `siteID:` label is unnecessary

```suggestion
        if case let Site.dotCom(_, bearerToken: token, _) = site {
```

---

Site note: I looked for a SwiftLint rule for this but did not find one that would have picked that up. At first, I browsed the [rules docs](https://realm.github.io/SwiftLint/rule-directory.html). Then, I tried with

```
cat WordPressKit/WordPressOrgRestApi.swift \
  | ./Pods/SwiftLint/swiftlint lint --quiet --enable-all-rules --use-stdin \
  | sort -v
```